app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def return_three_most_recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
Please ensure the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). All posts, comments, and likes should all have`foreign_key: 'author_id'` for a correct association 👍🏽 e.g:
`has_many :posts, foreign_key: 'author_id'`